n2y/main.py

```diff
@@ -61,67 +65,83 @@ def name_column_valid(raw_rows, name_column):
     first_row_flattened = simplify.flatten_database_row(raw_rows[0])
 
     def available_columns():
-        return filter(lambda c: isinstance(first_row_flattened[c], str),
-                      first_row_flattened.keys())
+        return filter(
+            lambda c: isinstance(first_row_flattened[c], str),
+            first_row_flattened.keys())
 
     # make sure the title column exists
     if name_column not in first_row_flattened:
-        print(f"Database does not contain the column \"{name_column}\". "
-              f"Please specify the correct name column using the --name-column flag.",
-              file=sys.stderr)
-        print("Available column(s): ",
-              # only show columns that have strings as possible options
-              ", ".join(available_columns()), file=sys.stderr)
-        return False
+        raise NotImplementedError("%s%s%s" % (
+            f"ERROR: Database Does Not Contain The Column \"{name_column}\".\n",
+            "Please Specify The Correct Name Column Using The --name-column Flag.\n",
+            # only show columns that have strings as possible options
+            "Available Column(s): " + ", ".join(available_columns())))
```

Comment:
Raising an exception displays the stack trace and looks like the program crashed. I don't think the program should "crash". Previous behavior was good. Display the problem and possible solution then exit with a non-zero return value.

Also, if we wanted to throw an exception, `NotImplementedError` is not the correct one here. See https://docs.python.org/3/library/exceptions.html for exception types.